### PR TITLE
Chore/tech 290

### PR DESCRIPTION
**Pull request by mendesmcmg**
# Description

- Changes color calls to new DS way

## Does this pull request close any open issues?

- [#tech-290](https://ribon.atlassian.net/browse/TECH-290?atlOrigin=eyJpIjoiMDA4MzJhOTIyMTZjNGRjYThmZjY0MzQ5MzY5ZThmYWYiLCJwIjoiaiJ9)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [x] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Test A

### CODE REVIEW

File: src/styles/theme.ts [hunk 1]
```diff
@@ -61,7 +54,7 @@ const theme: ThemeType = {
     defaultShadow05: "rgba(40, 36, 28, 0.05)",
     backgroundOverlay: "rgba(40, 36, 28, 0.60)",
     brand: brandColors,
-    neutral: neutralColors,
```
Comment (thaisribon) [inline src/styles/theme.ts:64]: why did you change to colorNeutral? i think the old one is better :/

Comment (mendesmcmg) [reply src/styles/theme.ts:64]: right, I'll change it back :)

**Review by thaisribon — APPROVED** [review]
